src/ctr_logging.c

```diff
@@ -257,6 +274,9 @@ int write_journald(int pipe, char *buf, ssize_t buflen)
 		if (name && writev_buffer_append_segment(dev_null, &bufv, container_name, name_len + NAME_EQ_LEN) < 0)
 			return -1;
 
+		if (writev_buffer_append_segment(dev_null, &bufv, syslog_identifier, syslog_identifier_len) < 0)
```

Comment:
No sure but i think we must check if `syslog_identifier` is valid before flushing. Maybe 
```c
if (syslog_identifier && writev_buffer_append_segment(dev_null, &bufv, syslog_identifier, syslog_identifier_len) < 0)
```

Reply:
I suppose we could but I intentionally made it so that syslog_identifier was always defined (either container name or ID). if it'd make you more comfortable I can update

Reply:
@haircommander oh it is completely fine then :) . Thanks for confirming.